### DISCUSSION TITLE
Self-hosting Simple Login with an additional mail server already running.

### DISCUSSION THREAD

**Discussion by SolninjaA** [simple-login/app#1628] [Q&A]
Hello, I currently have a mail server already running and I want to also self-host Simple Login. From what I can see, Simple Login uses the same ports as my mail server which gives me the impression it will cause a problem. I was wondering if there was a way to self-host both a mail server and Simple Login at the same time.

Thanks in advance!

**Comment by nguyenkims** [comment]
@SolninjaA as far as I know, this isn't possible unfortunately ...

**Reply by SolninjaA** [reply]
Hi @nguyenkims, thanks for replying. I had the idea of having a reverse proxy using different backends for each subdomain and having SimpleLogin’s DNS records on another domain. I tried this but I was having issues with SSL certificates, however, if I can get past that problem do you think it might work?

Thanks again!

**Reply by nguyenkims** [reply]
@SolninjaA unfortunately I'm not familiar with running another email server in parallel with SimpleLogin ...
